i18n/en.json

```diff
@@ -1020,15 +1020,15 @@
   },
   {
     "id": "api.command_mute.hint",
-    "translation": "[channel]"
+    "translation": "~[channel]"
   },
   {
     "id": "api.command_mute.name",
     "translation": "mute"
   },
   {
     "id": "api.command_mute.error",
-    "translation": "Could not find the channel {{.Channel}}."
+    "translation": "Could not find the channel {{.Channel}}. Please use the <a href=\"https://about.mattermost.com/default-channel-handle-documentation/\">channel handle</a> to identify channels."
```

Comment:
If I traced the code correctly, I think this translated bit of text is used as the `Message` of a `Post`, so I'm pretty sure you can just write this as Markdown.